public/electron.js

```diff
@@ -33,6 +33,8 @@ const gotTheLock = app.requestSingleInstanceLock()
 // Build menu
 
 const isMac = process.platform === 'darwin'
+const isWindows = process.platform === 'win32'
+const isNotMacOrWindows = !isMac && !isWindows
```

Comment:
Is this var rly needed at the end? It's only used once. Why not just use `!isMac && !isWindows` directly in the condition?

Reply:
I found it useful to abstract what "Is not mac or windows" means since the underlying check could become more complicated or possibly re-used in the future